tools/install-deps

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -xue
+
+if [[ "$OSTYPE" =~ "linux" ]]; then
+    if which apt-get; then
+        # Debian or Ubuntu
```

Comment:
On PCLinuxOS apt calls rpm instead of dpkg, so you could check for dpkg too to at least exclude PCLinuxOS:
`if [ $(which apt-get) ] && [ $(which dpkg) ]; then`